test/libyul/yulInterpreterTests/infinite_recursion.yul

```diff
@@ -0,0 +1,11 @@
+{
+  function f() {
+    f()
+  }
+  f()
```

Comment:
What about including a positive test?

```
{
    function f(x) {
        if lt(x, 512) {
            f(add(x, 1))
        }
    }
    f(0)
}
```

Reply:
Added a positive test but I'm afraid it didn't turn out exactly like above mainly because of how we count an execution "step" (yul statement).

More details here: https://github.com/ethereum/solidity/blob/3cd0b252206550f8aa1158eb3e78665ff77c5b54/test/tools/yulInterpreter/Interpreter.cpp#L193-L195